pkgs/development/tools/misc/semver-tool/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "semver-tool-${version}";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fsaintjacques";
+    repo = "semver-tool";
+    rev = version;
+    sha256 = "0lpwsa86qb5w6vbnsn892nb3qpxxx9ifxch8pw3ahzx2dqhdgnrr";
+  };
+
+  installPhase = ''
```

Comment:
Please add `runHook preInstall` and `runHook postInstall`. And also report the issue with path hardcoded in the install path.

Reply:
Oh, glad to see that's something we're caring about now :)